dbt/include/bigquery/macros/adapters.sql

```diff
@@ -53,8 +53,10 @@
 
     create or replace table {{ relation }}
       {% if config.get('contract', False) %}
-        {{ get_assert_columns_equivalent(sql) }}
+        {% do log('======== has a contract: ' ~ compiled_code) %}
```

Comment:
Why do we log for this adapter, but not for the others? I'm fine either way, but we should probably be consistent.

Reply:
I think that was leftover from debugging. I'll remove it. There was some confusion about the difference between "compiled_code" and "sql" in the macros (they're both context methods, so people weren't careful about using the passed in string...)